controller/article-controller.js

```diff
@@ -35,10 +35,22 @@ exports.getCommentsByArticleId = (req, res, next) => {
 exports.postComments = (req, res, next) => {
     const {username, body} = req.body
     const {article_id} = req.params
+
     checkArticleExists(article_id)
-    .then(()=>insertComments(username, body, article_id))
+    .then(() => insertComments(username, body, article_id))
     .then((comment)=>{
         res.status(201).send(comment)
     })
     .catch(next)
+}
+
+exports.patchArticle = (req, res , next) => {
+    const votes = req.body.inc_votes
+    const id = req.params.article_id
+    checkArticleExists(id)
```

Comment:
You shouldn't need to check if the article ID exists here. We should allow psql to throw an error in the model then we can make the check and use a custom error in there.